.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
```

Comment:
In cases like this, it is better to create tiny PR's. .gitignore doesn't have anything to do with this PR and a tiny change can be reviewed very quickly.

Reply:
Echo Marti's point. I am fine with leaving it as is for this PR. But going forward, if reviewers make a suggestion like this and there's no strong reason for not doing so, the right way to really address the comment is to revert the change, break it up, and send it for review in a separate PR.

Reply:
Keep in mind that having smaller and more focused PRs is really a good practice of separation of concerns. It would make your and reviewer's life much easier with focused changes because you want your reviewers to be very focused, and you don't want to deal with too many comments all at once.